linode/manager
# upcoming: [M3-7937] - Update Placement Groups detail and summaries

## Description 📝
This PR achieves two main goals:
- Consolidate summaries styles and their data (drawers, PG detail)
- Consolidate PG detail (summary + linodes list, no tabs)
- Remove "(`Affinity Type`)" appendages

## Changes  🔄
- Introduce new `DescriptionList` component for summaries (only used for PG, meant to be expanded)
  - component, test, storybook
- Implement `DescriptionList` in PG drawers and PG detail
- Remove tabs on Placement Group Details page
- Remove (`Affinity Type`)" appendages in
  - drawers
  - breadcrumb
- Update tests


## Preview 📷

| PG Detail | Create Drawer | Assign Drawer | Edit Drawer | Storybook |
| ------- | ------- | ------- | ------- | ------- |
| ![Screenshot 2024-04-05 at 09 08 17](https://github.com/linode/manager/assets/130582365/b7663d2a-66f1-4d73-9d30-a7834db16509) | ![Screen Shot 2024-04-05 at 09 09 37](https://github.com/linode/manager/assets/130582365/6e6aaa2b-427a-45f2-be23-54b34cae5470) | ![Screen Shot 2024-04-05 at 09 40 32](https://github.com/linode/manager/assets/130582365/5b7a337b-c3e6-4ee7-a96a-889eeeeca53f) | ![Screen Shot 2024-04-05 at 09 41 55](https://github.com/linode/manager/assets/130582365/77729ea7-ebd2-41aa-9437-4b1d1ae287e4) | ![localhost_6006__path=_docs_components-descriptionlist--documentation (1)](https://github.com/linode/manager/assets/130582365/1e0fa823-8936-48ee-a2eb-1ac591ff7196) |

## How to test 🧪

⚠️ **Note:** Copy is still WIP (main copy, errors, notices)

### Prerequisites
- Switch your environment to ALHPA and pull code locally
- Login with user `pg-user-1` (1Password)

### Verification steps
- Navigate to http://localhost:3000/placement-groups
  - Confirm create and edit drawers
- Navigate to http://localhost:3000/placement-groups/{id}
  - Confirm detail page with summary
  - Confirm assign drawer
- Start storybook: `yarn storybook`
  - Navigate to http://localhost:6006/?path=/docs/components-descriptionlist--documentation
  - Confirm `DescriptionList` behavior and styling via using the playground

## As an Author I have considered 🤔

*Check all that apply*
